addon/adapters/pouch.js

```diff
@@ -97,49 +104,55 @@ export default DS.RESTAdapter.extend({
     var plural = pluralize(recordTypeName);
 
     // check that we haven't already registered this model
+    var isSchemaNew = true;
     for (var i = 0, len = this._schema.length; i < len; i++) {
       var currentSchemaDef = this._schema[i];
       if (currentSchemaDef.singular === singular) {
-        return;
+        isSchemaNew = false;
+        break;
```

Comment:
Yes, the main problem is that the model name is the same, ( for me 'project' ) which means that the schema does not need to be recreated BUT needs to be applied to the new DB instance ( or else 'rel' is not defined ). That the goal of this code and the code below.